### PR TITLE
add additional steps to work around broken generator

### PR DESCRIPTION
**Pull request by nicogeburek**

### CODE REVIEW

File: tutorials/cp-cf-sapui5-local-debug/cp-cf-sapui5-local-debug.md [hunk 1]
```diff
@@ -3,8 +3,8 @@ title: Debug Your SAPUI5 App
 description: Debugging and logging are the nut and bolts to inspect your application. Learn how to debug and test your web apps with the Google Chrome Dev Tools.
 auto_validation: true
 time: 15
-tags: [ tutorial>beginner, topic>html5, topic>cloud, topic>javascript, products>sap-cloud-platform-for-the-cloud-foundry-environment, tutorial>free-tier]
-primary_tag: topic>sapui5
+tags: [ tutorial>beginner, programming-tool>html5, topic>cloud, programming-tool>javascript, products>sap-cloud-platform-for-the-cloud-foundry-environment, tutorial>free-tier]
```
Comment (MichaelCzcz) [inline tutorials/cp-cf-sapui5-local-debug/cp-cf-sapui5-local-debug.md:6]: Please update tags. products>sap-cloud-platform-for-the-cloud-foundry-environment -> products>sap-btp--cloud-foundry-environment

File: tutorials/sapui5-fiori-cf-display-data/sapui5-fiori-cf-display-data.md [hunk 2]
```diff
@@ -39,25 +39,34 @@ The routes are added to the `uimodule/webapp/manifest.json` file. The generator
...
-```JSON [3]
-{
-  "name": "Products",
-  "pattern": "",
-  "target": [
-    "TargetProducts"
-  ]
-},
-```
+1. As you can see in the log, there are two new files and one modified file. As the generator is only able to create boilerplate code, we have to make some modifications to the `uimodule/webapp/manifest.json` application descriptor.
```
Comment (MichaelCzcz) [inline tutorials/sapui5-fiori-cf-display-data/sapui5-fiori-cf-display-data.md:42]: Please don't say "we" when addressing the user, say "you" instead.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

File: tutorials/cp-cf-sapui5-local-debug/cp-cf-sapui5-local-debug.md (hunk 1 above)
Comment (nicogeburek) [reply tutorials/cp-cf-sapui5-local-debug/cp-cf-sapui5-local-debug.md]: done, thanks a lot

File: tutorials/sapui5-fiori-cf-display-data/sapui5-fiori-cf-display-data.md (hunk 2 above)
Comment (nicogeburek) [reply tutorials/sapui5-fiori-cf-display-data/sapui5-fiori-cf-display-data.md]: done, thanks a lot

**Pull request closed without merging**